pkg/apis/core/validation/validation.go

```diff
@@ -5750,20 +5758,20 @@ func validateContainerResize(newRequirements, oldRequirements *core.ResourceRequ
 			break
 		}
 	}
-	if memRestartPolicy == core.NotRequired || memRestartPolicy == "" {
+	if memRestartPolicy == core.ResizeRestartPolicyNotRequired || memRestartPolicy == "" {
```

Comment:
I'm confused - there are two names that mean the same thing - don't you need to test for both?  If my policy is set to PreferNoRestsart, I still want to execute this block, right?